frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -631,12 +672,20 @@ export const ClusterServiceVersionsPage: React.FC<ClusterServiceVersionsPageProp
     </p>
   );
 
-  const flatten = ({ clusterServiceVersions, subscriptions }) =>
-    [
-      ..._.get(clusterServiceVersions, 'data', []),
+  const flatten = ({ clusterServiceVersions, subscriptions, allClusterServiceVersions }) => {
+    const csvData = _.get(clusterServiceVersions, 'data', []);
+    const allCSVFiltered = _.get(allClusterServiceVersions, 'data', []).filter((el) => {
+      return !csvData.some((f) => {
+        return f.metadata.name === el.metadata.name;
+      });
+    });
+    return [
+      ...csvData,
+      ...allCSVFiltered,
       ..._.get(subscriptions, 'data', []).filter(
         (sub) =>
-          ['', sub.metadata.namespace].includes(props.namespace) &&
+          (['', sub.metadata.namespace].includes(props.namespace) ||
+            ['', sub.metadata.namespace].includes('openshift-operators')) &&
```

Comment:
We should never hard-code `openshift-operators`. (See my other comment about this.)